.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,22 +1,48 @@
 ---
-name: Bug Report
-about: Reporting issues with the VA API Platform
-
+name: 🔴 Bug Report
+about: For reporting issues with the VA API Platform
 ---
 
-## APIs Affected
+Important note: As this is a public forum, please do not include your VA API OAuth Client ID, VA API OAuth Client Secret, VA API key, or any Personally Identifiable Information (PII) or Protected Health Information (PHI) in your report.
```

Comment:
Maybe add a ⚠️ emoji or bold to help call this out a bit more.

Reply:
Great call. Added ⚠️ to call it out better.